src/applications/letters/containers/LetterList.jsx

```diff
@@ -26,12 +30,22 @@ export class LetterList extends React.Component {
   }
 
   componentDidMount() {
-    const { shouldUseLighthouse } = this.props;
+    const { letters, shouldUseLighthouse } = this.props;
     focusElement('h2#nav-form-header');
-    this.setState({
-      // eslint-disable-next-line -- LH_MIGRATION
-      LH_MIGRATION__options: LH_MIGRATION__getOptions(shouldUseLighthouse),
-    });
+    this.setState(
+      {
+        // eslint-disable-next-line -- LH_MIGRATION
+        LH_MIGRATION__options: LH_MIGRATION__getOptions(shouldUseLighthouse),
+      },
+      () => {
+        // Need updated LH_MIGRATION__options for correct download URL
+        this.props.getLettersPdfLinksWrapper(
+          // eslint-disable-next-line -- LH_MIGRATION
+          this.state.LH_MIGRATION__options,
+          letters,
+        );
+      },
+    );
```

Comment:
Opted to make the new call for PDFs as a callback function. I needed to wait for `LH_MIGRATION__options` to be settled to work correctly, so I kicked off the dispatch in a callback function.